CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changes
```

Comment:
These are all breaking changes and I will fix this up post acceptance prior to release.